lib/l2meter/configuration.rb

```diff
@@ -16,6 +16,7 @@ def initialize
       @output = $stdout
       @float_precision = 4
       @context = nil
+      @output_true_false_nil_values = false
```

Comment:
We should probably document this in the `README`.

Reply:
Ah yeah, of course, thanks for the reminder.

Reply:
See https://github.com/heroku/l2meter/pull/9/commits/f44bf73274ec9f829025a8b86c6b416be31ab172.